File: WebAPI/WebAPI/Models/PhieuNhapXuat/PhieuNhapXuatDto.cs
using System;
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using WebAPI.Models.ThongTinChiTietThietBi;

namespace WebAPI.Models.PhieuNhapXuat
{
	public class PhieuNhapXuatDto
	{
        public int Id { get; set; }
        public string Ma { get; set; }
        public DateTime NgayNhapXuat { get; set; }
        public string NhaCungCap { get; set; }
        public string NguoiDaiDien { get; set; }
        public int NhanVienId { get; set; }
        public int? SoLuong { get; set; }
        public decimal? TongTien { get; set; }
        [MaxLength(255)]
        public string GhiChu { get; set; }
        public int LoaiPhieu { get; set; }
        public List<ThongTinChiTietThietBiDto> ThongTinChiTietThietBiDtos { get; set; }
    }
}

File: WebAPP/src/app/models/PhieuNhapXuatDto.ts
export class PhieuNhapXuatDto {
  id: number;
  ma: string;
  ngayNhapXuat: Date;
  nhaCungCap: string;
  nguoiDaiDien: string;
  nhanVienId: number;
  soLuong: number | null;
  tongTien: number | null;
  ghiChu: string;
  loaiPhieu: number;
  thongTinChiTietThietBiDtos: ThongTinChiTietThietBiDto[] | null;
}

export class ThongTinChiTietThietBiDto {
  id: number | any;
  ma: string;
  thietBiYTeId: number;
  ngayNhap: Date | null;
  xuatXu: string;
  namSX?: number | null;
  hangSanXuat: string;
  tinhTrang: number | null;
  khoaId: number | null;
  nhanVienId: number | null;
  serial: string;
  model: string;
  giaTien: number | null;
  thoiGianBaoDuong: number | null;
  daXuat: boolean | null;
}
